kuma/wiki/admin.py

```diff
@@ -363,6 +364,30 @@ def slug_short(self, obj):
         return truncatechars(obj.slug, 25)
     slug_short.short_description = 'Slug'
 
+    def doc_short(self, obj):
+        """
+        Shorten document 'path (name)' representation in list view.
+
+        The important part is having an HTML break character such as a space,
+        so truncating paths to 25 characters after the locale, and long titles
+        at 25 characters.
+        """
+        doc = obj.document
+        if doc:
+            full_path = '/%s/docs/%s' % (doc.locale, doc.slug)
+            if len(full_path) <= 25:
```

Comment:
If you want to have consistent length between non-ASCII characters using `len(unicodedata.normalize(full_path, 'NFC'))` would help, assuming `full_path` is a Unicode string